### ISSUE TITLE
Quick Question

### ISSUE THREAD

**Issue by Rygar1976** [ianmarmour/nvidia-clerk#182]
Hi there, as ive never done anything like this, can I relist the steps so you can tell me if this is working as intended?

Download your file for windows.
Open CMD prompt and type:  set NVIDIA_CLERK_SKU=5438481700
Drag the file I downloaded into the CMD prompt window and drop
Now add the following to the command it pasted into the CMD Prompt window:
 -region=USA -model=3090
Hit Enter

I now get a repeating command that I believe is woring and just not finding anything for sale in my region before it hits the search time limit?  

So I leave the command window open and if it finds a 3090 for sale in my region, will it stop and do something to notify me, or do I just watch the command line screen?  Does it tell me where to find it, or give me a link?
Basically, if this is working right, what happens when it actually finds something, and how do I know it did so?

TYUTYTYTYTYTYTYTYT!

**Comment by Rygar1976** [comment]
Oops - apologies - i am reading on this now and now sure how to delete!  Gah...

**Comment by Rygar1976** [comment]
2

**Comment by Rygar1976** [comment]
Sorry
